acceptance-tests.md

```diff
@@ -146,14 +173,16 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
 
 1. **Track/report usage of the cluster**
     * _As an administrator of the cluster, I should be able to view daily, weekly and monthly reports of the cluster infrastructure utilization._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
```

Comment:
Do we need to check that XDMOD and ACM usage totals match, and if so how do we do that?  Some of this will need to be assigned to Rob.

Reply:
@computate I think this is an engineering task for you to do once XDMod info is available, but it does not have to gate acceptance tests.